test/jdk/sun/security/lib/cacerts/AmazonCACertConstants.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2017, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Since we aren't pushing this upstream, the copyright be Amazon.

Reply:
Using license here instead (will update the year to 2023): https://github.com/rgithubli/corretto-17/blob/94070ab18a1f14effcbfc9da2ebfa98ed57aa4d0/installers/linux/universal/rpm/build.gradle#L1-L20